code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -37,7 +37,8 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/antag_hud_name
 	/// If set to true, the antag will not be added to the living antag list.
 	var/soft_antag = FALSE
-
+	/// The battlecry this antagonist shouts when suiciding with C4/X4.
+	var/suicide_cry = ""
```

Comment:
I recommend making this a weird default so future antagonists do not forget to add it.

Reply:
I was going to leave it blank/null so any unset antags just use the default c4 suicide warcry, since it's not like it's a necessary feature.

Reply:
eh, fine with me